packages/stark-core/src/modules/error-handler/error-handlers.module.ts

```diff
@@ -0,0 +1,40 @@
+import { ErrorHandler, ModuleWithProviders, NgModule, Optional, SkipSelf } from "@angular/core";
+import { StarkErrorHandler } from "./handlers/stark-error-handler";
+
+@NgModule({
+})
+export class StarkErrorHandlerModule {
```

Comment:
Let's be consistent with the rest of our modules:
- the file name should be "error-handling.module"
- the class name should be "StarkErrorHandlingModule"
- the folder name of this module should be "error-handling"